### PR TITLE
python-dateutil -> 2.4.2

### PR DESCRIPTION
**Pull request by risicle**

### CODE REVIEW

**Comment by risicle** [comment]
hmm unrelated travis failure

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -2807,11 +2807,13 @@ let
...
 
   dateutil = buildPythonPackage (rec {
-    name = "dateutil-2.2";
+    name = "dateutil-${version}";
+    version = "2.4.2";
 
     src = pkgs.fetchurl {
       url = "http://pypi.python.org/packages/source/p/python-dateutil/python-${name}.tar.gz";
-      sha256 = "0s74ad6r789810s10dxgvaf48ni6adac2icrdad34zxygqq6bj7f";
+      md5 = "4ef68e1c485b09e9f034e10473e5add2";
```
Comment (domenkozar) [inline pkgs/top-level/python-packages.nix]: remove md5

**Comment by risicle** [comment]
Yup, that errored because "build log too long"

**Comment by domenkozar** [comment]
Did you use nox-review to test? it might break other packages

**Comment by risicle** [comment]
I have not. I have quite limited processing capability at the moment.

**Comment by FRidh** [comment]
Thanks for the PR @risicle . This update has been merged: e1520f85fded2c543427e6feaffbfb21ccab1f78

**Comment by domenkozar** [comment]
Cool!

**Pull request closed without merging**